frontend/src/components/sign/components/redirect-after-sign/redirect-after-sign.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect } from 'react';
+import { useSearchParams } from 'react-router-dom';
+
+import { StorageKey } from '@common/enums/app/storage-key.enum';
+import { Grid, CircularProgress } from '@mui/material';
+
+const RedirectAfterSign = (): React.ReactElement => {
+  const [searchParams] = useSearchParams();
+  const accessToken = searchParams.get('accessToken');
+  const refreshToken = searchParams.get('refreshToken');
+
+  useEffect(() => {
+    if (accessToken && refreshToken) {
+      localStorage.setItem(StorageKey.TOKEN, accessToken);
+      localStorage.setItem(StorageKey.REFRESH, refreshToken);
+    }
+    window.close();
+  }, []);
+  return (
```

Comment:
[Let it breathe](https://youtu.be/m7jfM9A6R-k?t=56) 
```suggestion
  }, []);

  return (
```